com.tencent.wemeet.yml

```diff
@@ -0,0 +1,65 @@
+app-id: com.tencent.wemeet
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: wemeetapp
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --socket=pulseaudio
+- --device=all
+- --allow=devel
+- --talk-name=org.freedesktop.Notifications
+- --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: wemeet
+    buildsystem: simple
+    build-commands:
+      - install -D wemeetapp.sh /app/bin/wemeetapp
+      - install -D apply_extra -t /app/bin
+      - install -D /usr/bin/desktop-file-edit -t /app/bin/
+      - install -D /usr/bin/ar -t /app/bin
+      - ARCH_TRIPLE=$(gcc --print-multiarch) &&
+        install -D /usr/lib/${ARCH_TRIPLE}/libbfd-*.so -t /app/lib
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - ar x *.deb $(ar t *.deb|grep data)
+          - tar -xf data.* && rm *.deb data.*
+          - mv usr export
+          - cp -a opt/wemeet/icons export/share/
+          - ls -d -1 export/share/icons/hicolor/* | 
+            xargs -i cp -a {}/{mimetypes,apps}
+          - FLATPAK_ID=com.tencent.wemeet
+          - desktop-file-edit 
+            --set-icon="$FLATPAK_ID" 
+            --set-key="Exec" --set-value="wemeetapp %U" 
+            export/share/applications/*.desktop
+          - rename "wemeetapp" "$FLATPAK_ID" 
+            export/share/icons/hicolor/*/*/*.png 
+            export/share/applications/*.desktop
+
+      - type: script
+        dest-filename: wemeetapp.sh
+        commands:
+          - export XDG_SESSION_TYPE=x11
+          - export QT_QPA_PLATFORM=xcb
```

Comment:
you can add these to the `--finish-args` instead. Even though  I dont think they are necessary since only `x11` is allowed